python_modules/dagster/dagster/_cli/api.py

```diff
@@ -100,7 +101,7 @@ def _metrics_polling_interval(
     try:
         return float(
             dagster_run.tags.get(
-                "dagster/run_metrics_polling_interval_seconds",
+                f"{HIDDEN_TAG_PREFIX}run_metrics_polling_interval_seconds",
```

Comment:
most of the other tags that start with HIDDEN_TAG_PREFIX are defined in that tags.py module, you could continue that convention here as well